feat: atlantis import (#2783)
runatlantis/atlantis: runatlantis.io/docs/server-side-repo-config.md

## Use Cases
Here are some of the reasons you might want to use a repo config.

> ### Requiring PR Is Approved Before Apply or Import

Reviewer: If we add a future subcommand then this title may turn into

```suggestion
### Requiring PR Is Approved Before Apply or Import or NewSubcommand
```

Can we instead make this simply

```suggestion
### Requiring PR Is Approved Before an applicable subcommand
```

Or something similarly generic?

repos:
- id: github.com/myorg/myrepo
  apply_requirements: [mergeable]
  import_requirements: [mergeable]
```

See [Command Requirements](command-requirements.html) for more details.

> ### Repos Can Set Their Own Apply or Import Requirements

Reviewer: Same comment https://github.com/runatlantis/atlantis/pull/2783/files#r1054041699

Reviewer: ```suggestion
### Repos Can Set Their Own Apply an applicable subcommand
```
